test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -272,6 +274,28 @@ TEST(QuicPlatformTest, QuicNotification) {
   EXPECT_TRUE(notification.HasBeenNotified());
 }
 
+TEST(QuicPlatformTest, QuicCertUtils) {
+  bssl::UniquePtr<X509> x509_cert =
+      TransportSockets::Tls::readCertFromFile(TestEnvironment::substitute(
+          "{{ test_rundir }}/test/extensions/transport_sockets/tls/test_data/san_dns_cert.pem"));
+  int len = i2d_X509(x509_cert.get(), nullptr);
```

Comment:
optional: add comment describing what this is doing, e.g. "Convert X509 to DER"

Reply:
oh, never mind, I see @htuch already assigned @PiotrSikora to look at that aspect (thanks!)

Reply:
done